packages/mui-utils/src/elementAcceptingRef/elementAcceptingRef.test.tsx

```diff
@@ -20,94 +22,83 @@ describe('elementAcceptingRef', () => {
   });
 
   describe('acceptance when not required', () => {
-    let rootNode: HTMLElement;
-
-    function assertPass(element: any, { shouldMount = true } = {}) {
+    async function assertPass(element: any, { shouldMount = true } = {}) {
       function testAct() {
         checkPropType(element);
         if (shouldMount) {
           // TODO: replace with React 18 implementation after https://github.com/testing-library/react-testing-library/issues/1216 is closed.
```

Comment:
By using `waitFor` as [suggested](https://github.com/testing-library/testing-library-docs/issues/1255#issuecomment-1557558510) in https://github.com/testing-library/react-testing-library/issues/1216, can we remove this comment?

Reply:
I missed this. Comments removed.